org/markAsMergeOnGreen.ts

```diff
@@ -0,0 +1,67 @@
+import { danger } from "danger"
+import { IssueComment } from "github-webhook-event-types"
+
+// The shape of a label
+interface Label {
+  id: number
+  url: string
+  name: string
+  description: string
+  color: string
+  default: boolean
+}
+
+/** If a comment to an issue contains "Merge on Green", apply a label for it to be merged when green. */
+export const rfc10 = async (issueComment: IssueComment) => {
+  const issue = issueComment.issue
+  const comment = issueComment.comment
+  const api = danger.github.api
+
+  // Only look at PR issue comments, this isn't in the type system
+  if (!(issue as any).pull_request) {
+    return console.log("Not a Pull Request")
+  }
+
+  // Don't do any work unless we have to
+  const keywords = ["merge on green", "merge on ci green"]
```

Comment:
would love to have this require referencing a bot, sorta like `@CocoaPodsBot merge on green` - would be much more explicit